web/src/components/storage/DASDPage.tsx

```diff
@@ -40,6 +41,10 @@ export default function DASDPage() {
         <DASDTable />
         <DASDFormatProgress />
       </Page.Content>
+
+      <Page.Actions>
+        <Page.Back>{_("Close")}</Page.Back>
```

Comment:
I agree, but navigating back is also "dangerous", currently it goes to the form dialog if you press back after trying to activate a disk which is not the expected behavior, therefore, I prefered to specify the path and force to navigate to the entry point of the route..

Reply:
What do you mean by "dangerous"?

Reply:
I think it's not dangerous at all. Just a bit messy. In any case, I had a chat with @teclator and understood the _problem_ with the Page.Back action. A _problem_ that still there because the user can use the browser back. 

So, we agreed to keep it as a navigation action and address the back/cancel _problems_ in other PR.

BTW, the _problem_ could be easily solve by [replacing](https://reactrouter.com/en/main/hooks/use-navigate#optionsreplace) the route when navigating back from or submitting a form since, maybe, we want the user reaching a form **explicitly**